x-pack/plugins/ml/public/components/full_time_range_selector/index.tsx

```diff
@@ -12,17 +12,44 @@ import { IndexPattern } from 'ui/index_patterns';
 import { EuiButton } from '@elastic/eui';
 import { setFullTimeRange } from './full_time_range_selector_service';
 
+export interface TimeRange {
```

Comment:
Is it worth moving out this `TimeRange` interface to the top-level common `types` folder? Is this something which might be used outside the context of the full time range selector control?